bin/runn.js

```diff
@@ -1,19 +1,29 @@
 #!/usr/bin/env node
 
 let program = require('commander');
-let runn = require("../src/runn");
- 
+let runn = require('../src/runn');
+let chalk = require('chalk');
+let figlet = require('figlet');
+
 program
   .version('0.1.0')
   .arguments('<cmd> [set]')
   .action(function(cmd, set) {
-    if(set) 
-        runn.saveCommand(cmd, set);
-    else
-        runn.runCommand(cmd);
+    if (set) runn.saveCommand(cmd, set);
+    else runn.runCommand(cmd);
   });
 
-  
 program.parse(process.argv);
 
-if(process.argv.length < 3) program.help();
+
+
+if (process.argv.length < 3) {
+  console.log(
+    chalk.blue(
+      figlet.textSync('RUNN', {
+        font: 'Cyberlarge'
+      })
+    )
+  );
```

Comment:
Just noticed program.help(); is missing.  We need to show figlet + help when someone just enter the command (without parameters)

Reply:
![snip](https://user-images.githubusercontent.com/38534289/72684056-537f1700-3b03-11ea-9435-8be43ed44394.PNG)

this is how currently working. 
who should i change this to

Reply:
![ss](https://user-images.githubusercontent.com/38534289/72684118-c38d9d00-3b03-11ea-8c87-70b6b375aea2.PNG)

Reply:
Hey.. Simply the outputs of two case of first image need to be same

Reply:
ahhh i got it , working on it .

Reply:
![1212121](https://user-images.githubusercontent.com/38534289/72816459-2e6adf80-3c8e-11ea-9be8-14c43124d91b.PNG)